### PR TITLE
Database

### PR DESCRIPTION
**Pull request by arnaldoaparicio**
Working database.

### CODE REVIEW

File: app/models/invoice_item.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class InvoiceItem < ApplicationRecord
-  belongs_to :items
-  belongs_to :invoices
+  belongs_to :item
```
Comment (tjroeder) [inline app/models/invoice_item.rb:2]: Pluralization is the bane of my existence good catch.

File: db/migrate/20220103224548_create_transactions.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class CreateTransactions < ActiveRecord::Migration[5.2]
   def change
     create_table :transactions do |t|
-      t.references :invoices, foreign_key: true
-      t.integer :credit_card_number
+      t.references :invoice, foreign_key: true
+      t.string :credit_card_number
```
Comment (tjroeder) [inline db/migrate/20220103224548_create_transactions.rb:5]: Good call on changing this to a String. As an Int we can't fit in all 16 characters. As well we shouldn't have to do any math on the CC number either.

**Review by tjroeder — APPROVED** [review]
Great update with the Rake file!

**Review by WadeNaughton — APPROVED** [review]
removing pluralization is a good catch

**Pull request merged**